sys/riscv/riscv/trap.c

```diff
@@ -368,8 +379,15 @@ data_abort(struct trapframe *frame, int usermode)
 	return;
 
 fatal:
+#if __has_feature(capabilities)
+	if (frame->tf_scause == EXCP_STORE_AMO_CAP_PAGE_FAULT) {
+		panictype = " capability";
```

Comment:
I'd personally put the space at the end, since the "capability" is associated with page fault not fatal.